lib/module.js

```diff
@@ -74,6 +74,13 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+const builtinModules = Object.keys(NativeModule._source)
+  .filter(NativeModule.nonInternalExists);
+
+Object.freeze(builtinModules);
+
+module.exports.builtinModules = builtinModules;
+
```

Comment:
Since `Module` is exported as `module.exports` you can just bolt on `builtinModules` to
```js
Module.builtinModules = builtinModules
```